ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -73,6 +73,18 @@ class PropertySpec: QuickSpec {
 				mutableProperty = nil
 				expect(signalCompleted).to(beTruthy())
 			}
+
+			it("should not deadlock on recursive value access") {
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+				var property = MutableProperty(0)
+
+				property <~ producer
+				property.producer.start(next: { _ in
+					let test = property.value
```

Comment:
Can you add a flag that ensures this gets called when we expect? (It'd be easy for this test to pass if the block were just never invoked somehow.)